brightray/browser/browser_main_parts.cc

```diff
@@ -168,7 +168,9 @@ BrowserMainParts::~BrowserMainParts() {
 void OverrideAppLogsPath() {
 #if defined(OS_WIN)
   std::wstring app_name = base::UTF8ToWide(GetApplicationName());
-  std::wstring log_path = L"%HOMEDRIVE%%HOMEPATH%\\AppData\\Roaming\\";
+  std::wstring home = std::wstring(_wgetenv(L"HOMEDRIVE"));
+  std::wstring drive = std::wstring(_wgetenv(L"HOMEPATH"));
+  std::wstring log_path = home + L"\\" + drive + L"\\AppData\\Roaming\\";
   std::wstring app_log_path = log_path + app_name + L"\\logs";
```

Comment:
Hm, in order to be consistent I think we should do

```cpp
base::FilePath path;
PathService::Get(brightray::DIR_APP_DATA, &path);
return path + L"\\logs";
```

Then we don't have to worry about environment variables and the behavior RE appdata location is consistent.

Reply:
If there already is an abstraction like that, then awesome, let's use it instead. 👍

Reply:
@alespergl Yeah and it uses `SHGetKnownFolderPath` internally 😄

Reply:
... let's just make sure any failure of `PathService::Get` is handled and we don't end up with `\logs` as the final path. 😃

Reply:
Heh, good point, `Get` returns a success boolean so should be safe to check that

Reply:
can you just add a string to a `FilePath` like that?? and what would be returned in the instance that `PathService::Get` fails?